### PR TITLE
{math}[foss/2018b] Clp v1.17.3

### PR DESCRIPTION
**Pull request by tstrempel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by tstrempel** [comment]
https://github.com/easybuilders/easybuild-easyconfigs/pull/10035 needed beforehand

File: easybuild/easyconfigs/c/Clp/Clp-1.17.3-foss-2018b.eb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+sources = [SOURCE_TGZ]
+checksums = ['a13bf54291ad503cf76f5f93f2643d2add4faa5d0e60ff2db902ef715c094573']
+
+builddependencies = [
+    ('Doxygen', '1.8.14'),
+]
+
+dependencies = [
+    ('OpenBLAS', '0.3.1'),
+    ('ScaLAPACK', '2.0.2', '-OpenBLAS-0.3.1'),
```
Comment (smoors) [inline easybuild/easyconfigs/c/Clp/Clp-1.17.3-foss-2018b.eb]: @tstrempel those 2 deps are loaded by default by the `foss` toolchain, so you can remove them.

Comment (tstrempel) [reply easybuild/easyconfigs/c/Clp/Clp-1.17.3-foss-2018b.eb]: @smoors I didn't know, that the math libraries are implicitly added to the module file. I'll update my pull request.

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 4 out of 4 (1 easyconfigs in this PR)
node352.hydra.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/4aefbf5a98e86399d1f97fc6076ecbb9 for a full test report.

**Comment by smoors** [comment]
Going in, thanks @tstrempel!

**Pull request merged**